docs/metrics.asciidoc

```diff
@@ -277,3 +282,173 @@ You can filter and group by these dimensions:
 * `span.subtype`: The sub-type of the span, for example `mysql` (optional)
 
 --
+
+
+[float]
+[[metrics-micrometer]]
+=== Custom metrics using Micrometer
+
+The Elastic APM Java agent lets you use the popular metrics collection framework https://micrometer.io/[Micrometer] to track custom application metrics.
+
+Some use cases for tracking custom metrics from your application include monitoring performance-related things like cache statistics, thread pools, or page hits.
+However, you can also track business-related metrics such as revenue and correlate them with performance metrics.
+Metrics registered to a Micrometer `MeterRegistry` are aggregated in memory and reported every <<config-metrics-interval, `metrics_interval`>>.
+Based on the metadata about the service and the timestamp, you can correlate metrics with traces.
+However, you won't be able to attribute a value to a specific transaction or to a user.
+If you'd like to do that, have a look at the <<api-transaction-add-tag>> API method instead.
```

Comment:
You mean- using the same labels for the event and the meter? Try to make it more explicit.

Reply:
No, what I was trying to say is that if you want to correlate to individual events don't use metrics or micrometer at all. Instead, add labels to the transaction. But then you either have to do 100% sampling or account for the missing events.

Reply:
Then I suggest explaining this a bit clearer.

Reply:
I rephrased it. Hope it's clearer now.

Reply:
It's perfect